client/app/lib/constants/sharedConstants.ts

```diff
@@ -78,3 +79,5 @@ export const DEFAULT_LOCALE = process.env.DEFAULT_LOCALE ?? 'en';
 
 export const DEFAULT_TIME_ZONE =
   process.env.DEFAULT_TIME_ZONE ?? 'Asia/Singapore';
+
+export const NUM_CELL_CLASS_NAME = 'text-right';
```

Comment:
You might want to just add this to the table cell component related file.

Reply:
if I do that, there will be linter error of that text 'text-right' being repeated more than some number of times (if I remembered correctly, the threshold is 6)

I have also considered an option of putting this NUM_CELL_CLASS_NAME inside Table component, however I think the current one that is here is much cleaner, and also might be useful for any other purposes since this is sharedConstants..